app/services/edition_filter.rb

```diff
@@ -41,6 +44,22 @@ def allowed_sort?(sort)
     SORT_KEYS.flat_map { |item| [item, "-#{item}"] }.include?(sort)
   end
 
+  def access_limited_scope(scope)
+    scope.where(
+      "access_limit_id IS NULL" + " OR " +
+      "(
+         access_limits.limit_type = 'primary_organisation' AND
+         #{TAG_CONTAINS_QUERY % { tag: 'primary_publishing_organisation' }}
+       )" + " OR " +
+      "(
+         access_limits.limit_type = 'all_organisations' AND
+         (#{TAG_CONTAINS_QUERY % { tag: 'primary_publishing_organisation' }} OR
+          #{TAG_CONTAINS_QUERY % { tag: 'organisations' }})
+       )",
+      value: user.organisation_content_id,
```

Comment:
Do you know if this works ok if the value of organisation_content_id is nil?

Reply:
It does, but I've added a couple of extra tests to make sure.